app/helpers/view_helper.rb

```diff
@@ -30,7 +30,18 @@ def govuk_back_link_to(url = :back, body = 'Back')
       body = 'Back to application'
     end
 
-    render GovukComponent::BackLink.new(text: body, href: url, classes: classes)
+    render GovukComponent::BackLink.new(
+      text: body,
+      href: url,
+      classes: classes,
+    )
+  end
+
+  def govuk_breadcrumbs(breadcrumbs)
+    render GovukComponent::Breadcrumbs.new(
+      breadcrumbs: breadcrumbs,
+      hide_in_print: true,
+    )
```

Comment:
Added a helper so that we can pass common options into GovukComponent::Breadcrumbs.

Reply:
I think govuk-components already defines this helper (https://github.com/DFE-Digital/govuk-components/blob/master/app/helpers/govuk_components_helper.rb). I think this is OK but it could cause confusion because it's overriding the helper from the gem.

Reply:
The helper only aliases the component, so using the same name and passing default options options seems fine to me. Would a comment help?

Reply:
Oh wait, we’re doing a bit more than that, as we’re taking `breadcrumbs` value directly. Hmmm.

Reply:
Updated name of helper to `breadcrumbs`.